internal/impl/sql/output_sql_insert.go

```diff
@@ -113,8 +113,9 @@ type sqlInsertOutput struct {
 	builder squirrel.InsertBuilder
 	dbMut   sync.RWMutex
 
-	useTxStmt   bool
-	argsMapping *bloblang.Executor
+	useTxStmt     bool
+	argsMapping   *bloblang.Executor
+	argsConverter func([]any) []any
```

Comment:
I can create an alias NP.

> would it not be cleaner to store the driver string and then use an if statement where you call

Maybe? I am probably over indexing on performance. Instead of comparing a string everytime, we can just directly jump to the function.

Reply:
Haha, OK. I didn't think about performance implications :)

Reply:
Added an alias for the type